app/controllers/top_controller.rb

```diff
@@ -1,6 +1,10 @@
 class TopController < ApplicationController
   def show
     @database_memos = DatabaseMemo.all.includes(:data_source, schema_memos: :table_memos).sort_by(&:display_order)
-    @favorite_tables = TableMemo.where(id: current_user.favorite_tables.pluck(:table_memo_id))
+    if current_user
+      @favorite_tables = TableMemo.where(id: current_user.favorite_tables.pluck(:table_memo_id))
+    else
+      @favorite_tables = []
+    end
```

Comment:
I want to hide favorite tables without login user.

![image](https://user-images.githubusercontent.com/50920/33838326-9a2bb6ca-ded2-11e7-815e-98ac541d6866.png)